docker-compose.yml

```diff
@@ -1,20 +1,43 @@
 version: "3.1"
 services:
   auth2:
-    image: kbase/kb_auth2:${TRAVIS_BRANCH:-latest}
+    image: kbase/kb_auth2:latest
     ports:
       - "8080:8080"
-    depends_on: ["ci-mongo"]
+    environment:
+      - KB_DEPLOYMENT_CONFIG=/kb/deployment/conf/deployment.cfg
+    command:
```

Comment:
Document this is overriden by mini_kb

Reply:
I included a large explanatory comment at the start of the docker-compose file

Reply:
See the start of the file. This compose file has a lot more comments in it now.